web_src/css/base.css

```diff
@@ -989,14 +989,6 @@ table th[data-sortt-desc] .svg {
   box-shadow: 0 0 0 1px var(--color-secondary) inset;
 }
 
-.emoji {
-  font-size: 1.25em;
-  line-height: var(--line-height-default);
-  font-style: normal !important;
-  font-weight: var(--font-weight-normal) !important;
-  vertical-align: -0.075em;
-}
-
 .emoji img {
```

Comment:
```suggestion
/* for "image" emojis like ":git:"  ":gitea:" and ":github:" (see CUSTOM_EMOJIS config option) */
.emoji img {
```

![image](https://github.com/user-attachments/assets/e4525697-e682-4ec4-85ad-e0ff78c219ee)